### ISSUE TITLE
Variables should be camel cased

### ISSUE THREAD

**Issue by proidiot** [stuphlabs/pullcord#88]
Variables should be camel cased (brought up by @michaelbeam in #70).

**Comment by geekthattweaks** [comment]
@proidiot think I may take a swing at this one.

**Comment by proidiot** [comment]
So I was "working" on #142 , which would probably include this work, but I hadn't got to this bit yet, and frankly that issue should have been split up into many smaller pieces from the outset anyhow. The sheer volume of linter errors from gometalinter could certainly go by much faster with some help. I just now went ahead and created #147 to finish up my current work on documenting exported symbols (i'll probably rename my working branch). Most of these errors are just the result of coding style choices I made before I was understood many aspects of "the go way" and I just never got around to correcting them as I became more comfortable with the language. However, there are also some nastier bits here and there, but I figure it is at least worth knocking out the low-hanging fruit first.
